client/elements/menus/settings-menu.html

```diff
@@ -385,6 +393,12 @@ <h3 class="nerdy-row">Select this to remember these settings next time you visit
               this.shadowRoot.querySelector('#chinese_lookup_menu').addEventListener('value-changed', (e) => {
                   this._chineseLookupChanged(e.detail.value);
               });
+              this.shadowRoot.querySelector('#pali_lookup_menu').addEventListener('selected-item-changed', () => {
+                  this.paliSelectedItemChanged = true;
```

Comment:
This is much better, but I'm thinking - couldn't we just call `this._showToast(dictName)` here and get rid of the additional properties? I think `dictName` can be retrieved from the passed event, like this:

```
this.shadowRoot.querySelector('#chinese_lookup_menu').addEventListener('selected-item-changed', (**event**) => {
    console.dir(event); // The dictionary name should be somewhere in this object.
    this.chineseSelectedItemChanged = true;
});
```

Reply:
I suppose we could. The Dictname has to be converted for the showToast but that's only a minor string conversion. Will get on it and upload soon.

Reply:
?? You said :+1: but you dit not approve the pull request ???
So I'm not sure if I was clear or not: the above code (calling the function directly at the EventListener) did not work. Therefore I did not push it to the branch.

Reply:
Sorry. Approving now.